src/cogent3/parse/genbank.py

```diff
@@ -4,7 +4,10 @@
 from cogent3.core.info import Info
 from cogent3.core.moltype import get_moltype
 from cogent3.parse.record import FieldWrapper
-from cogent3.parse.record_finder import DelimitedRecordFinder, LabeledRecordFinder
+from cogent3.parse.record_finder import (
```

Comment:
this is a black / isort issue. As this change is also made by the linter PR, remove it from here otherwise it will be a conflict.